stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/BaseConnectedIntegrationTest.kt

```diff
@@ -57,11 +59,11 @@ internal open class BaseConnectedIntegrationTest : BaseDomainTest() {
         chatDomain = chatDomainImpl
         chatDomainImpl.retryPolicy = NoRetryPolicy()
         chatDomainImpl.repos.insertUsers(data.userMap.values.toList())
-        chatDomainImpl.errorEvents.observeForever(
-            EventObserver {
+        chatDomainImpl.scope.launch {
```

Comment:
Need to launch new coroutine otherwise collecting will block since `StateFlow` is never finishing flow objects